tests/dataDriven_test.js

```diff
@@ -0,0 +1,34 @@
+let data = new DataTable(["url", "statusCode"]);
```

Comment:
Please move all the related "DataTable" code above the scenario it is for. So line 1-9 should be moved above line 25. We like to keep this information next to each other.